<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>unlock_customer_account</class>
 <widget class="QWidget" name="unlock_customer_account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Unlock customer &amp; account</string>
  </property>
  <widget class="QLabel" name="address_output">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>135</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="email_output">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>189</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="unlock_button">
   <property name="geometry">
    <rect>
     <x>56</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unlock</string>
   </property>
  </widget>
  <widget class="QLabel" name="email_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>190</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="character_limit">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>395</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="phone_text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>105</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="created_output">
   <property name="geometry">
    <rect>
     <x>208</x>
     <y>220</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text_input">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>20</y>
     <width>44</width>
     <height>12</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="account_text">
   <property name="geometry">
    <rect>
     <x>197</x>
     <y>161</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_text">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>248</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Status</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="reason_message_input">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>286</y>
     <width>371</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Why this customer/account is being unlocked?</string>
   </property>
  </widget>
  <widget class="QLabel" name="created_text">
   <property name="geometry">
    <rect>
     <x>151</x>
     <y>218</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Created in</string>
   </property>
  </widget>
  <widget class="QLabel" name="birthday_output">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>162</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="address_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>132</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text_output">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>104</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_text">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>77</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_button">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>16</y>
     <width>90</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_output">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>107</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="name_output">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>79</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="account_output">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>162</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="id_text">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>218</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_output">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>107</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>44</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="cpf_field_input">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>16</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="birthday_text">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>161</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Birthday</string>
   </property>
  </widget>
  <widget class="QLabel" name="id_output">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>228</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_output">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cpf_field_input</tabstop>
  <tabstop>search_button</tabstop>
  <tabstop>reason_message_input</tabstop>
  <tabstop>unlock_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
